db/rdbms/src/main/java/io/camunda/db/rdbms/read/service/ProcessDefinitionInstanceVersionStatisticsDbReader.java

```diff
@@ -7,26 +7,65 @@
  */
 package io.camunda.db.rdbms.read.service;
 
-import io.camunda.db.rdbms.sql.columns.SearchColumn;
+import io.camunda.db.rdbms.read.domain.ProcessDefinitionInstanceVersionStatisticsDbQuery;
+import io.camunda.db.rdbms.sql.ProcessDefinitionMapper;
+import io.camunda.db.rdbms.sql.columns.ProcessDefinitionInstanceVersionStatisticsSearchColumn;
 import io.camunda.search.clients.reader.ProcessDefinitionInstanceVersionStatisticsReader;
 import io.camunda.search.entities.ProcessDefinitionInstanceVersionStatisticsEntity;
 import io.camunda.search.query.ProcessDefinitionInstanceVersionStatisticsQuery;
 import io.camunda.search.query.SearchQueryResult;
 import io.camunda.security.reader.ResourceAccessChecks;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ProcessDefinitionInstanceVersionStatisticsDbReader
     extends AbstractEntityReader<ProcessDefinitionInstanceVersionStatisticsEntity>
     implements ProcessDefinitionInstanceVersionStatisticsReader {
 
-  public ProcessDefinitionInstanceVersionStatisticsDbReader() {
-    super(new SearchColumn[] {});
+  private static final Logger LOG =
+      LoggerFactory.getLogger(ProcessDefinitionInstanceVersionStatisticsDbReader.class);
+
+  private final ProcessDefinitionMapper processDefinitionMapper;
+
+  public ProcessDefinitionInstanceVersionStatisticsDbReader(
+      final ProcessDefinitionMapper processDefinitionMapper) {
+    super(ProcessDefinitionInstanceVersionStatisticsSearchColumn.values());
+    this.processDefinitionMapper = processDefinitionMapper;
   }
 
   @Override
   public SearchQueryResult<ProcessDefinitionInstanceVersionStatisticsEntity> aggregate(
       final ProcessDefinitionInstanceVersionStatisticsQuery query,
       final ResourceAccessChecks resourceAccessChecks) {
-    // Not implemented yet
-    return SearchQueryResult.empty();
+
+    final var dbSort =
+        convertSort(
+            query.sort(),
+            ProcessDefinitionInstanceVersionStatisticsSearchColumn.PROCESS_DEFINITION_ID);
```

Comment:
💭 This is the discriminator column in case of otherwise identical values. Isn't the `VERSION` the discriminator here? 🤔